packages/react/src/components/HorizontalRule/__stories__/HorizontalRule.stories.js

```diff
@@ -52,10 +52,14 @@ storiesOf('HorizontalRule', module)
           <div className="bx--row bx--no-gutter">
             <div className="bx--col">
               <HorizontalRule
-                style={select('style', styles, '')}
-                size={select('size', sizes, '')}
-                contrast={select('contrast', contrasts, '')}
-                weight={select('weight', weights, '')}
+                style={select('style', styles, styles.solid)}
```

Comment:
Love these changes.

I'm not sure how the rest of the team feels about this, but changes like this if we were to create a separate pull request we could get it in and review it a lot faster than including it within an already opened but larger pull request like this one.

Just something to think about as we move forward.

cc: @jeffchew @kennylam

Reply:
I thought since this was a minor change I could just pull it in but I can see it interfering with the main goal of this PR - I'll make sure to separate it out next time

Reply:
You're good! 😄 I just think we can get it in faster outside. Same outcome I think. It's more of a comment to consider down the road.